spec/models/orchestration_stack/retirement_management_spec.rb

```diff
@@ -1,5 +1,5 @@
 describe "Service Retirement Management" do
-  let(:user) { FactoryGirl.create(:user) }
+  let!(:user) { FactoryGirl.create(:user_miq_request_approver, :userid => 'admin') }
```

Comment:
`userid` is sequenced by the factory and shouldn't need to be set in any of these.

Reply:
I agree with you that it shouldn't be set here, but our factory doesn't do what seeding does. Am I allowed to change the factory?

Reply:
Is the `userid` relevant to the tests?  They pass without it.

Reply:
"...but this conversation seemed unresolved..." 

I fixed it, it's resolved.